packages/@aws-cdk/aws-apigateway/lib/model.ts

```diff
@@ -0,0 +1,13 @@
+export interface IModel {
+    readonly modelId: string;
+}
+
+export class EmptyModel implements IModel {
```

Comment:
Please add docs and provide a link to official documentation describing the keywords 'Empty' and 'Error'.